awsconfigs/common/ack-sagemaker/kustomization.yaml

```diff
@@ -0,0 +1,49 @@
+# cluster scoped ack installation 
+
+bases:
+- ./base/default/
+
+patches:
+- patch : |-
+    - op: add
+      path: "/metadata/labels"
+      value: 
+        rbac.authorization.kubeflow.org/aggregate-to-kubeflow-edit: "true"
+  target:
+    kind: ClusterRole
+    name: ack-sagemaker-controller
+- patch : |-
+    - op: add
+      path: "/metadata/annotations"
+      value:
+        eks.amazonaws.com/role-arn : $(ACK_SAGEMAKER_OIDC_ROLE)
+  target:
+    kind: ServiceAccount
+    name: ack-sagemaker-controller
+  target:
+    kind: Deployment
+    name: ack-sagemaker-controller
+    namespace: ack-system
+
+patchesStrategicMerge:
+- deployment_env_patch.yaml
+
+# Read and replace params
+
+configMapGenerator:
+- name: ack-install-parameters
+  envs:
+  - params.env
+generatorOptions:
+  disableNameSuffixHash: true
+vars:
+- name: ACK_SAGEMAKER_OIDC_ROLE
+  objref:
+    kind: ConfigMap
+    name: ack-install-parameters
```

Comment:
change we change this configmap name to `ack-sagemaker-controller-parameters`?

Reply:
- it would have been good to move this under an overlay named kubeflow
- can we rename `ack-sagemaker` directory name to sagemaker-operator? because that is the official product name
sagemaker-controller is also okay